spec/high_voltage/configuration_spec.rb

```diff
@@ -48,7 +48,20 @@
     end
   end
 
-  describe '#action_caching_layout=' do
+  describe '#action_caching=false' do
+    it 'does not display a deprecation warning' do
+      allow(ActiveSupport::Deprecation).to receive(:warn)
+
+      HighVoltage.configure do |config|
+        config.action_caching = false
+      end
+
+      expect(ActiveSupport::Deprecation).not_to have_received(:warn)
+                                                .with(HighVoltage::Configuration::CACHING_DEPRECATION_WARNING)
```

Comment:
Line is too long. [110/80]<br>Place the . on the previous line, together with the method call receiver.<br>Use 2 (not 42) spaces for indenting an expression spanning multiple lines.